### ISSUE TITLE
Package not putting files separately in Azure storage container for each tenant

### ISSUE THREAD

**Issue by hamees-allshore** [Azure-OSS/azure-storage-php-adapter-laravel#3]
I have installed this package in a web application that is running of Azure with Azure Storage containers.
The app is storing all files on the root of the container while it should have created a separate folder for each tenant and then have placed files in them.

I have configured everything correctly but seems that the storage adapter is not considering the suffix parameter set by the multi-tenancy for laravel package.

_config/filesystems.php_
```
// ...
'azure' => [
            'driver' => 'azure-storage-blob',
            'connection_string' => env('AZURE_STORAGE_CONNECTION_STRING'),
            'container' => env('AZURE_STORAGE_CONTAINER'),
        ],
// ...
```

_config/tenancy.php_
```
// ...
'filesystem' => [
        'suffix_base' => 'tenant',
        'disks' => [
            'local',
            'public',
            'azure',
        ],
// ...
```

Reference: #https://tenancyforlaravel.com/docs/v3/tenancy-bootstrappers/#filesystem-tenancy-boostrapper

**Comment by hamees-allshore** [comment]
#2 This pull request may fix this issue.

Tested the fix for my project.

**Comment by brecht-vermeersch** [comment]
Released version 1.1.0 with the fix. Can you confirm that this fixes your issue?
